library/Icingadb/Widget/Detail/EventDetail.php

```diff
@@ -141,19 +143,24 @@ protected function assembleNotificationEvent(NotificationHistory $notification)
             $users = $users->execute();
             /** @var ResultSet $users */
 
-            $this->addHtml(
-                new UserList($users),
-                (new ShowMore($users, Links::users()->addParams([
-                        'notification_history.id' => bin2hex($notification->id)
-                    ]), sprintf(t('Show all %d recipients'), $notification->users_notified)))
-                    ->setBaseTarget('_next')
-            );
+            $notifiedUsers[] = new UserList($users);
+            $notifiedUsers[] = (new ShowMore(
+                $users,
+                Links::users()->addParams(['notification_history.id' => bin2hex($notification->id)]),
+                sprintf(t('Show all %d recipients'), $notification->users_notified)
+            ))->setBaseTarget('_next');
         }
+
+        $this->add(ObjectDetailExtensionHook::injectExtensions([
+            100 => $pluginOutput,
```

Comment:
The output section is at `0`. `100` is the graph section. Take a look at `BaseExtensionHook::BASE_LOCATIONS` and subtract 1000 of each.